Lib/Protocols/IdRegister.pas

```diff
@@ -277,9 +272,7 @@ implementation
   IdSASLOTP,
   IdSASLPlain,
   IdSASLSKey,
-  IdSASLUserPass,
   IdSASL_CRAM_MD5,
-  IdSASL_CRAM_SHA1,

```

Comment:
Removing `IdSASL_CRAM_SHA1` causes an error in Delphi, as `TIdSASLCRAMSHA1` is being registered in Delphi but not in FPC.  So either `IFDEF` this unit or fix the registration of `TIdSASLCRAMSHA1` for FPC.